portal_pages/templates/portal_pages/marketplace_entry_page_add.html

```diff
@@ -40,9 +40,12 @@
             <br />
             {{ logo_form.file.label_tag }}<br />
             {{ logo_form.file }}
-            {{ logo_form.file.errors }}
+            {{ logo_form.non_field_errors }}
```

Comment:
This change won't be necessary if the cleaning is done via clean_<fieldname> rather than form-level clean.